source/lovelace/changelog.markdown

```diff
@@ -183,12 +183,12 @@ description: "Changelog of the Lovelace UI."
 - ⚠️ [sensor card]: Removed configs `height`, `line_color` and `line_width`
 - ⚠️ [gauge card]: Renamed configuration `title` to `name`
 - ⚠️ [alarm panel card]: Renamed configuration `title` to `name`
-- ⚠️ [glance card]: `tap_action` and `hold_action` configurations changed. See docs.
-- ⚠️ [entity button card]: `tap_action` and `hold_action` configurations changed. See docs.
-- ⚠️ [picture card]: `tap_action` and `hold_action` configurations changed. See docs.
-- ⚠️ [picture elements card]: `tap_action` and `hold_action` configurations for elements changed. See docs.
-- ⚠️ [picture entity card]: `tap_action` and `hold_action` configurations changed. See docs.
-- ⚠️ [picture glance card]: `tap_action` and `hold_action` configurations changed. See docs.
+- ⚠️ [glance card]: `tap_action` and `hold_action` configurations changed. See documentation.
+- ⚠️ [entity button card]: `tap_action` and `hold_action` configurations changed. See documentation.
+- ⚠️ [picture card]: `tap_action` and `hold_action` configurations changed. See documentation.
+- ⚠️ [picture elements card]: `tap_action` and `hold_action` configurations for elements changed. See documentation.
+- ⚠️ [picture entity card]: `tap_action` and `hold_action` configurations changed. See documentation.
+- ⚠️ [picture glance card]: `tap_action` and `hold_action` configurations changed. See documentation.
```

Comment:
I decided against changing "historical" documents...

Reply:
In general, we don't mind doing that. Reasoning:

- It allows us to enforce certain things (The Textlint adjustments prevent the use of "docs" in the future).
- It doesn't harm the context or contents in any way for this change.

Reply:
The blog posts are currently exempted from the Textlint, mainly because we include commits in there, which often contain spelling issues. Correcting those would divert from the actual commit message.